ansible_collections/juniper/device/plugins/connection/pyez.py

```diff
@@ -601,6 +601,8 @@ def open_configuration(self, mode, ignore_warn=None):
         if self.config is None:
             if mode not in CONFIG_MODE_CHOICES:
                 raise AnsibleError("Invalid configuration mode: %s" % mode)
+            if mode != 'ephemeral' and ephemeral_instance is not None:
+                self.fail_json(msg='configuration mode ephemeral is required')
```

Comment:
Change the message to "Ephemeral instance is specified while the mode is not ephemeral. Specify the mode as 'ephemeral' or do not specify the instance."

Reply:
Fixed

Output:
********
        }
    },
    "msg": "Ephemeral instance is specified while the mode is not ephemeral. Specify the mode as ephemeral or do not specify the instance."
}